monitoring/ping.go

```diff
@@ -183,29 +188,38 @@ func (c *pingChecker) checkNodesRTT(nodes []serf.Member, client *serf.RPCClient)
 }
 
 // storePingInHDR is used to store ping RoundTrip values in HDR Histograms in memory
-func (c *pingChecker) storePingInHDR(pingRttStats int64, node serf.Member) error {
-	nodeTTLMapInterface, exists := c.rttStats.Get(node.Name)
+func (c *pingChecker) storePingInHDR(pingroundtripLatency int64, node serf.Member) error {
+	nodeTTLMapInterface, exists := c.roundtripLatency.Get(node.Name)
 	if !exists {
-		c.rttStats.Set(node.Name,
+		c.roundtripLatency.Set(node.Name,
 			hdrhistogram.New(pingRoundtripMinimum.Nanoseconds(),
 				pingRoundtripMaximum.Nanoseconds(),
 				pingRoundtripSignificativeFigures),
 			statsTTLPeriod)
-		nodeTTLMapInterface, _ = c.rttStats.Get(node.Name)
+		nodeTTLMapInterface, _ = c.roundtripLatency.Get(node.Name)
 	}
 
-	nodeTTLMap := nodeTTLMapInterface.(*hdrhistogram.Histogram)
+	nodeTTLMap, ok := nodeTTLMapInterface.(*hdrhistogram.Histogram)
```

Comment:
I have a same comment here, in Go there is no need to communicate the type of the variable via variable name, because Go is a statically typed language. Instead it's better to communicate the intent of the variable:

So `nodeTTLMap` should be `nodeLatencies` and `nodeTTLMapInterface` should a temporary variable `n` because it does not carry any meaning and is used in one line only